docs/getting-started/linux-installer.md

```diff
@@ -1,391 +1,5 @@
-> The official Splunk documentation for this page is [Install on Linux](https://docs.splunk.com/Observability/gdi/opentelemetry/install-linux.html). For instructions on how to contribute to the docs, see [CONTRIBUTING.md](../CONTRIBUTING#documentation.md).
+The official Splunk documentation for this page is [Install the Collector for Linux with the installer script](https://docs.splunk.com/Observability/gdi/opentelemetry/install-linux.html). 
```

Comment:
Since this link includes instructions for both the installer script and the deployment modules (ansible, chef, etc), should it be changed to something more generic like:
```suggestion
The official Splunk documentation for this page is [Install the Collector for Linux](https://docs.splunk.com/Observability/gdi/opentelemetry/install-linux.html). 
```